experiments/base_experiment.py

```diff
@@ -585,7 +585,14 @@ def _step(self, data, step):
                 .item()
             )
         else:
-            grad_norm = 0.0  # meaningless placeholder
+            grad_norm =grad_norm = (
+                torch.nn.utils.clip_grad_norm_(
+                    self.model.parameters(),
+                    float('inf'),
```

Comment:
I think it would be simpler to have something like `self.cfg.training.clip_grad_norm if self.cfg.training.clip_grad_norm is not None else float('inf')` and avoid the `else` statement